lib/store.js

```diff
@@ -1,24 +1,38 @@
+import { lru } from 'tiny-lru'
+import { DEFAULT_OPTIONS } from './constants.js'
 export class Store {
-  constructor(timeWindowMs) {
+  constructor(
+    timeWindowMs,
+    intervalTimeExpiredKeysEvictionMs,
+    cacheSize = DEFAULT_OPTIONS.cacheSize
```

Comment:
don't apply defaults here. apply them only when parsing the options in the plugin initialization. this class shouldn't know anything about default values